core_lib/src/interface/timelinecells.cpp

```diff
@@ -49,6 +49,7 @@ TimeLineCells::TimeLineCells(TimeLine* parent, Editor* editor, TIMELINE_CELL_TYP
     setAttribute(Qt::WA_OpaquePaintEvent, false);
 
     connect(mPrefs, &PreferenceManager::optionChanged, this, &TimeLineCells::loadSetting);
+
```

Comment:
It doesn't really matter, but if you are making the changes anyway could you delete this trailing line.